### PR TITLE
Update dither per sample

### PR DESCRIPTION
**Pull request by defensem3ch**

### CODE REVIEW

**Comment by jkotlinski** [comment]
if the dither setting is per-sample, it should also be stored in the .kit.settings file.
you can see how it is done for volume and pitch in loadKitSettings() and saveKitSettings()

File: src/main/java/kitEditor/KitEditor.java [hunk 1]
```diff
@@ -262,6 +263,8 @@ private void onSpinnerChanged() {
             compileKit();
         }
         samplePicker.setSelectedIndex(index);
+        System.out.print(sample.getDither());
```
Comment (jkotlinski) [inline src/main/java/kitEditor/KitEditor.java:266]: please remove debug printout ;)

Comment (defensem3ch) [reply src/main/java/kitEditor/KitEditor.java:266]: > please remove debug printout ;)

oops! yes, of course!

**Comment by defensem3ch** [comment]
> looks like a fine change!
> if the dither setting is per-sample, it should also be stored in the .kit.settings file.
> you can see how it is done for volume and pitch in loadKitSettings() and saveKitSettings()

sure, I will take a look!

**Review by jkotlinski — APPROVED** [review]

**Comment by jkotlinski** [comment]
thank you for the contribution!

**Pull request merged**